terracumber_config/tf_files/SUSEManager-Head-build-validation-NUE.tf

```diff
@@ -0,0 +1,1285 @@
+//}
+
+module "base_core" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-bv-head-"
+  use_avahi   = false
+  domain      = "mgr.suse.de"
+  images      = [ "sles12sp4o", "sles12sp5o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "sles15sp4o", "sles15sp5o", "slemicro51-ign", "slemicro52-ign", "slemicro53-ign", "slemicro54-ign", "slemicro55-ign", "almalinux9o", "centos7o", "libertylinux9o", "oraclelinux9o", "rocky8o", "rocky9o", "ubuntu2004o", "ubuntu2204o", "debian11o", "debian12o", "opensuse155o" ]
+
+  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+    additional_network = "192.168.43.0/24"
+  }
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//module "base_arm" {
+//  providers = {
+//    libvirt = libvirt.overdrive3
+//  }
+//
+//  source = "./modules/base"
+//
+//  cc_username = var.SCC_USER
+//  cc_password = var.SCC_PASSWORD
+//  name_prefix = "suma-bv-head-"
+//  use_avahi   = false
+//  domain      = "mgr.suse.de"
+//  images      = [ "opensuse154armo", "opensuse155armo" ]
+//
+//  mirror = "minima-mirror-ci-bv.mgr.suse.de"
+//  use_mirror_images = true
+//
+//  testsuite = true
+//
+//  provider_settings = {
+//    pool        = "ssd"
+//    bridge      = "br1"
+//  }
+//}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "srv"
+  image              = "slemicro55-ign"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:a1"
+    memory             = 40960
+    vcpu               = 10
+    data_pool          = "ssd"
+  }
+  runtime = "podman"
+  container_repository = "registry.opensuse.org/systemsmanagement/uyuni/master/containers/uyuni"
+  helm_chart_url = "oci://registry.opensuse.org/systemsmanagement/uyuni/master/charts/uyuni/server"
+
+  server_mounted_mirror          = "minima-mirror-ci-bv.mgr.suse.de"
+  java_debugging                 = false
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
+  disable_download_tokens        = false
+  disable_auto_bootstrap         = true
+  ssh_key_path                   = "./salt/controller/id_rsa.pub"
+  from_email                     = "root@suse.de"
+
+  //server_additional_repos
+
+}
+
+module "proxy" {
+  source             = "./modules/proxy"
```

Comment:
I think that instead of deploy a proxy module, we need to deploy a regular sle micro vm.

The reason about it, is that this module is preparing a regular Proxy, not a containerized proxy. So, anything done by salt states for this proxy is not necessary for a containerized proxy.

Instead, we want a regular sle micro 5.5 and have some salt states installing it from the correct registry and the correct container.

So, somehow what is described here https://github.com/SUSE/spacewalk/issues/19280#issuecomment-1887147700

Hope it make sense for you too: @cbosdo @aaannz

Reply:
As mentioned in the issue, we will not use a proxy initially and add it later once we are later in the milestones and also have the new sumaform module for a containerized proxy.